cves/kernel/CVE-2016-6156.yml

```diff
@@ -171,8 +174,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: no information is provided.
```

Comment:
Again, it's nitpicky but capitalizing the first letter would maintain consistent formatting.